systems/crusher_quick_start_guide.rst

```diff
@@ -1444,6 +1444,64 @@ The above log messages indicate the type of image required by each device, given
 
 ----
 
+Floating-Point (FP) Atomic Operations and Coarse/Fine Grained Memory Allocations
+--------------------------------------------------------------------------------
+
+The Crusher system, equipped with CDNA2-based architecture MI250X cards, offers a coherent host interface that enables advanced memory and unique cache coherency capabilities.
+The AMD driver leverages the Heterogeneous Memory Management (HMM) support in the Linux kernel to perform seamless page migrations to/from CPU/GPUs.
+This new capability comes with a memory model that needs to be understood completely to avoid unexpected behavior in real applications. For more details, please visit the previous section.
+
+AMD GPUs can allocate two different types of memory locations: 1) Coarse grained and 2) Fine grained.
+
+**Coarse grained** memory is only guaranteed to be coherent outside of GPU kernels that modify it, enabling higher performance memory operations. Changes applied to coarse-grained memory by a GPU kernel are  only visible to the rest of the system (CPU or other GPUs) when the kernel has completed. A GPU kernel is only guaranteed to see changes applied to coarse grained memory by the rest of the system (CPU or other GPUs) if those changes were made before the kernel launched.
+
+**Fine grained** memory allows CPUs and GPUs to synchronize (via atomics) and coherently communicate with each other while the GPU kernel is running, allowing more advanced programming patterns. The additional visibility impacts the performance of fine grained allocated memory.
+The fast hardware-based Floating point (FP) atomic operations available on MI250X are assumed to be working on coarse grained memory regions; when these instructions are applied to a fine-grained memory region, they will silently produce a no-op. To avoid returning incorrect results, the compiler never emits hardware-based FP atomics instructions by default, even when applied to coarse grained memory regions. Currently, users can use the `-munsafe-fp-atomics` flag to suggest to the compiler that hardware-based FP atomics should be emitted.
```

Comment:
Another space is needed here to format correctly. Currently, the paragraph starting on 1459 is just added to the "Fine grained" sub-section, which isn't intended.